edit-hopper.php

```diff
@@ -35,9 +35,19 @@ public function __construct() {
 		}
 
 		function init($screen) {
+			$box_title = "Edit something";
+                        switch ($screen) {
+                            case "post":
+                                $box_title = "Edit Posts";
+                                break;
+                            case "page":
+                                $box_title = "Edit Pages";
+                            default:
+                                break;
+                        }
 			add_meta_box(
 		        'edit_hopper_box',
-		        __( 'Edit Pages', 'edit_hopper_textdomain' ),
+		        __( $box_title, 'edit_hopper_textdomain' ),
```

Comment:
That sounds like a much better idea.

On Thu, Mar 26, 2015, 9:08 PM Dylan Hildenbrand <notifications@github.com>
wrote:

> In edit-hopper.php
> <https://github.com/Dilden/Edit-Hopper/pull/1#discussion_r27272186>:
>
> >  			add_meta_box(
> >  		        'edit_hopper_box',
> > -		        __( 'Edit Pages', 'edit_hopper_textdomain' ),
> > +		        __( $box_title, 'edit_hopper_textdomain' ),
>
> Right. It should work just fine for pages and posts. The only thing I can
> see being a problem, is later on down the road when we add custom post
> types and options to show it on those as well. Those could be created with
> dashes in the name which would show up in the meta box title. We could
> probably query WP, and find the title of the custom post type and drop that
> in instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dilden/Edit-Hopper/pull/1/files#r27272186>.
>